packages/cli/src/error/profile_error.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * The aws profile error.
+ */
+export class ProfileError extends Error {
```

Comment:
```suggestion
export class InvalidCredentialError extends Error {
```

this might happen without profile being involved.